### ISSUE TITLE
Client doesn't load/work in browser

### ISSUE THREAD

**Issue by gtarcea** [mekwall/koa-ws#7]
Hi,

The default client available at /koaws.js isn't load in the browser due to the require statements included. It looks like the client code needs to be browserified. The browser displays an error on the first require statement (require('util')).

Could a browserified version of the client be served by default?

**Comment by mekwall** [comment]
Yeah. For now you'll have to include it in your webpack bundle or browserify it yourself.

I am currently working on providing a pre-built client. Hopefully I will have it done soon!

**Comment by Errandonea** [comment]
I include it in my webpack bundle for load the client on the browser. I have a webpack configuration, where i have the babel extension, for translate to JavaScript pure.

My problem is the next: I throw the client and I get this error in the chrome consol:

Websocket connetion to  'ws://koaws.js/' failed: Error in connection establishment: net::ERR_NAME_NOT_RESOLVERD 
 
Any ideas?
